examples/seismic/model.py

```diff
@@ -168,21 +168,21 @@ def demo_model(preset, **kwargs):
         # By default, the top part of the domain has 1.5 km/s,
         # and the bottom part of the domain has 2.5 km/s.
         shape = kwargs.pop('shape', (101, 101))
-        spacing = kwargs.pop('spacing', tuple([10. for _ in shape]))
+        spacing = kwargs.pop('spacing', tuple([1. for _ in shape]))
         origin = kwargs.pop('origin', tuple([0. for _ in shape]))
         dtype = kwargs.pop('dtype', np.float32)
         nbpml = kwargs.pop('nbpml', 10)
-        ratio = kwargs.pop('ratio', 2)
-        vp_top = kwargs.pop('vp_top', 1.5)
-        qp_top = kwargs.pop('qp_top', 10000.)
-        vs_top = kwargs.pop('vs_top', 0. * vp_top)
-        qs_top = kwargs.pop('qs_top', 0.)
-        rho_top = kwargs.pop('rho_top', 1.)
-        vp_bottom = kwargs.pop('vp_bottom', 2.5)
-        qp_bottom = kwargs.pop('qp_bottom', 10000.)
-        vs_bottom = kwargs.pop('vs_bottom', 0. * vp_bottom)
-        qs_bottom = kwargs.pop('qs_bottom', 0.)
-        rho_bottom = kwargs.pop('qs_bottom', 2.5/1.5)
+        ratio = kwargs.pop('ratio', 3)
+        vp_top = kwargs.pop('vp_top', 1.6)
+        qp_top = kwargs.pop('qp_top', 40.)
+        vs_top = kwargs.pop('vs_top', 0.4)
+        qs_top = kwargs.pop('qs_top', 30.)
+        rho_top = kwargs.pop('rho_top', 1.3)
+        vp_bottom = kwargs.pop('vp_bottom', 2.2)
+        qp_bottom = kwargs.pop('qp_bottom', 100.)
```

Comment:
same, seems low and unrealistic

Reply:
There's nothing unrealistic about these values. If we wish to benchmark with a different physical case that's something that can be considered though.